src/core_modules/capture-core/components/WidgetEventEdit/EditEventDataEntry/editEventDataEntry.epics.js

```diff
@@ -190,3 +193,23 @@ export const requestDeleteEventDataEntryEpic = (action$: InputObservable, store:
             return startDeleteEventDataEntry(eventId, params);
         }));
 
+export const startCreateNewAfterCompletingEpic = (
+    action$: InputObservable, store: ReduxStore, { querySingleResource, history }: ApiUtils) =>
+    action$.pipe(
+        ofType(
+            actionTypes.START_CREATE_NEW_AFTER_COMPLETING,
+            newEventWidgetActionTypes.START_CREATE_NEW_AFTER_COMPLETING,
+        ),
+        flatMap((action) => {
+            const { isCreateNew, enrollmentId } = action.payload;
+            return from(querySingleResource({ resource: 'tracker/enrollments', id: enrollmentId })).pipe(
```

Comment:
Hey,
Does it make sense to remove the call to the `tracker/enrollments` API endpoint? All the ids needed to build the page URL (orgUnitId, programId, teiId) are already in the components and can be passed to the `START_CREATE_NEW_AFTER_COMPLETING` action and then to the `startCreateNewAfterCompletingEpic` epic. 

I created a [patch](https://github.com/dhis2/capture-app/compare/DHIS2-13801...example-remove-endpoint-call?expand=1) to showcase one way how this can be done. WDYT?